### ISSUE TITLE
Issues with Code Along Video

### ISSUE THREAD

**Issue by MysticRiver** [learn-co-curriculum/html5-video-embed-code-along#6]
- There  should be a text based version of the video. Not everyone wants to watch a video and start and pause repeatedly.
- The video uses the code short cut git co, however that does not work in the Learn IDE. The equivalent code is git checkout

**Comment by matbalez-zz** [comment]
@MysticRiver we're going to enable the shortcut git commands in the Learn IDE, good catch

I also think you're right about the text based version of the lesson. @AnnJohn @pletcher

**Comment by AnnJohn** [comment]
yes, correct. thanks for raising this; we'll keep this issue open until we've added the text.

**Comment by DakotaLMartinez** [comment]
Hey y'all!

@curiositypaths Wanted to add that we've also had quite a few students have issues with this code along due to problems with IDE disconnects while trying to import the video files. Because this happens in a temporary directory, it's also triggering a reset and forcing the student to basically start from scratch.
